src/seedcase_sprout/core/extract_resource_properties.py

```diff
@@ -1,46 +1,66 @@
-from pathlib import Path
+import polars as pl
 
-from frictionless import describe
-from frictionless.resources import JsonResource
+import seedcase_sprout.core.check_datapackage as cdp
```

Comment:
```suggestion
from seedcase_sprout.core.sprout_checks.check_properties import check_resource_properties
```

We don't call `check_datapackage` outside of the Sprout specific `check_properties`

Reply:
Using Sprout specific checks here will fail bc the properties are incomplete.

```
  +-+---------------- 1 ----------------
    | seedcase_sprout.core.check_datapackage.check_error.CheckError: Error at `$.description` caused by `required`: 'description' is a required property
    +---------------- 2 ----------------
    | seedcase_sprout.core.check_datapackage.check_error.CheckError: Error at `$.name` caused by `required`: 'name' is a required property
    +---------------- 3 ----------------
    | seedcase_sprout.core.check_datapackage.check_error.CheckError: Error at `$.path` caused by `required`: 'path' is a required property
    +---------------- 4 ----------------
    | seedcase_sprout.core.check_datapackage.check_error.CheckError: Error at `$.title` caused by `required`: 'title' is a required property
```

Reply:
And as far as I understand, there’s not currently a way to ignore these using the Sprout checks. Correct me, if I'm wrong, @martonvago :)

Reply:
I’ve removed the check 🔥